.circleci/config.yml

```diff
@@ -81,8 +81,9 @@ jobs:
 
   integration_test:
     executor:
-      name: hmpps/node
-      tag: << pipeline.parameters.node-version >>
+      name: hmpps/node_redis
```

Comment:
Adding Redis here and in `docker-compose` to use 'real' Redis instead of default in-memory store for integration test environment.